setup.py

```diff
@@ -66,8 +66,8 @@ def create_edisgo_path():
         'geoplot': ['geopandas >= 0.9.0', 'contextily', 'descartes'],
         'examples': ['jupyter'],
         'dev': ['pytest', 'sphinx_rtd_theme'],
-        'full': ['geopandas >= 0.9.0', 'contextily', 'descartes', 'jupyter', 'pytest',
-                 'sphinx_rtd_theme']
+        'full': ['geopandas >= 0.9.0', 'contextily', 'descartes', 'jupyter', 'jupyterlab',
+                 'plotly', 'pytest', 'dash', 'sphinx_rtd_theme']
```

Comment:
When installing this environment and running some code I'm getting an error telling me that `jupyter_dash` is missing. So please add `jupyter_dash` as requirement.